codes/c/chapter_array_and_linkedlist/array.c

```diff
@@ -76,37 +93,41 @@ int main() {
     printf("数组 arr = ");
     printArray(arr, size);
 
-    int nums[5] = { 1, 3, 2, 5, 4 };
+    int nums[5] = { 1, 3, 3, 5, 4 };

```

Comment:
Please do not modify the test case

Reply:
> All the code languages are consistent in terms of the test cases, so please do not modify them.
I might need to close the pr cause I forgot to consider the compatibility with the content. Btw, thanks for replying!